### DISCUSSION TITLE
Wait for previous test to finish before going to the next test

### DISCUSSION THREAD

**Discussion by cyborgdennett** [bxparks/AUnit#101] [Q&A] [answered]
Hello AUnit community,

I am trying to test an arduino operating system using AUnit.

For this I use Epoxyduino EEPROM library, which stores the eeprom to a local file.

The problem I encouter is:
The tests run in parrallel and this ruins the EEPROM file.

What I would like to do is:
Run the test one after another.

Is there a way to do this? or needs to be a feature request?

**Comment by bxparks** [comment, marked as answer]
Tests should run serially by default. You need to do extra work to make them run concurrently, so I'm confused by your post. Maybe you can add additional details about your script, your Makefile, your *.ino file, etc.

**Reply by cyborgdennett** [reply]
It was my bad, I coded something badly ><
